packages/flutter/test/gestures/scale_test.dart

```diff
@@ -171,12 +219,15 @@ void main() {
     updatedFocalPoint = null;
     expect(updatedScale, 1.0);
     updatedScale = null;
+    expect(updatedRotation, 0.0);
+    updatedRotation = null;
 
     // We are done
     tester.route(pointer3.up());
     expect(didStartScale, isFalse);
     expect(updatedFocalPoint, isNull);
     expect(updatedScale, isNull);
+    expect(updatedRotation, isNull);
     expect(didEndScale, isTrue);
     didEndScale = false;
     expect(didTap, isFalse);
```

Comment:
Rather than adjusting existing tests, please create new tests (which can be based on copies of existing tests) to test the rotation logic.